site/content/installation.md

```diff
@@ -1,38 +1,40 @@
 
-To install Anaconda, simply go to the [Anaconda Download page](https://www.anaconda.com/distribution/#download-section), and click Download. We recommend the suggested defaults (i.e., install for "Just Me", use the default destination folder, and do not add Anaconda to your PATH environment). With these settings, you will NOT need administrator privileges to install Anaconda on your Urban computer.
-
-Once you have successfully installed Anaconda, you can open the Anaconda Navigator. This is a graphical interface that lets you launch applications to run Python code. To launch the Anaconda Navigator, search for it from the Start menu. 
-
-For more information, see the getting started guides for [Anaconda](https://docs.anaconda.com/anaconda/user-guide/getting-started/) and [Anaconda Navigator](https://docs.anaconda.com/navigator/getting-started/). 
+To confirm Python and `pip` (a package management tool) are both installed correctly, open a command prompt or terminal and run the following in the command line:
+`python --version` and `pip --version`.
 
 ## Choosing a Code Editor  
 
-There are a number of different code editors or integrated development environments (IDEs) for Python. The most popular ones among users at Urban include the following: 
+There are a number of different code editors or integrated development environments (IDEs) for Python. We recommend using [Visual Studio Code](https://code.visualstudio.com/) (or VS Code), which can be installed  from the [official website](https://code.visualstudio.com/download). VS Code can be used for standard .py files, as well as for Quarto and IPython notebooks (.qmd and .ipynb files).
 
-* [Jupyter Notebook](https://jupyter.org/) comes installed with Anaconda, and is similar to RMarkdown. Jupyter notebooks are web-based applications that let you edit and run blocks of code along with markdown text, equations, and visualizations in a single notebook file. 
-* [Spyder](https://www.spyder-ide.org/) also comes installed with Anaconda and is most similar to RStudio for R. It offers editing, analysis, and debugging support for Python. 
-* [Visual Studio Code](https://code.visualstudio.com/) (or VS Code) can be installed through Anaconda, and includes support for debugging, syntax highlighting, code completion, and more. It also offers thousands of extensions (including [Python-specific extensions](https://code.visualstudio.com/docs/languages/python), a [Quarto extension](https://quarto.org/docs/tools/vscode.html), and an [AWS extension](https://aws.amazon.com/visualstudiocode/)).  
+VSCode includes support for debugging, syntax highlighting, code completion, and more. It also offers thousands of extensions (including [Python-specific extensions](https://code.visualstudio.com/docs/languages/python), a [Quarto extension](https://quarto.org/docs/tools/vscode.html), and an [AWS extension](https://aws.amazon.com/visualstudiocode/)).  
```

Comment:
To-do: I will go through this process and make sure this isn't missing anything needed to get Python code running in VSCode in .py, .qmd or .ipynb files.